docs/designers-developers/designers/menu-item.md

```diff
@@ -0,0 +1,151 @@
+# MenuItem
+
+MenuItem is a component which renders a button intended to be used in combination with the DropdownMenu Component.
+
+![An image of a MenuItem being highlighted inside of a DropdownMenu component](https://wordpress.org/gutenberg/files/2018/11/MenuItem.png)
+
+1. MenuItem
```

Comment:
Not sure why do we need this?

Reply:
Hey! Thanks! I'm excited to get started. Thank you for taking a look and leaving feedback. Greatly appreciated. 

> Not sure why do we need this?

Are you referring to the image that I included? The thinking was that it could be helpful for folks to have a visual included of how the component is used. I'd love to hear your additional thoughts here.

Reply:
Oh sorry for the confusion, the README is great :). I was actually talking about this particular line `1. MenuItem`. It felt useless?

Reply:
The thinking was that the `<1. MenuIte>`  label corresponds to the 1 in the image diagram. It's less helpful in this example but in other examples there could be multiple items in an image that we want to clarify and this labeling would help. We could remove it for this example because there is only one item, but it could be useful to establish the pattern for other examples.

What do you think?

Reply:
Oh I totally missed the diagram while looking at the diff 🙃 

So yes, it's definitely worth keeping. Thanks for the clarification.